lib/node_modules/@stdlib/stats/base/dists/negative-binomial/logpmf/benchmark/benchmark.js

```diff
@@ -21,9 +21,9 @@
 // MODULES //
 
 var bench = require( '@stdlib/bench' );
-var ceil = require( '@stdlib/math/base/special/ceil' );
-var uniform = require( '@stdlib/random/base/uniform' );
-var randu = require( '@stdlib/random/base/randu' );
+var uniform	= require( '@stdlib/random/base/uniform' );
+var discreteUniform	= require( '@stdlib/random/base/discrete-uniform' );
```

Comment:
```suggestion
var uniform = require( '@stdlib/random/base/uniform' );
var discreteUniform = require( '@stdlib/random/base/discrete-uniform' );
```

The spacing needs to be fixed